src/almanack/repository_processing.py

```diff
@@ -0,0 +1,116 @@
+"""
+This module processes GitHub repositories
```

Comment:
This is a good idea and a good point! I've created sub-directories, since I think `git_operations.py` and `repository_processing.py` still have different use cases, and the separation of the two allow for future work to be added too. Please let me know if you think these sub-directories do a good job, or if they need further editing!